### PR TITLE
Create settings.py

### PR DESCRIPTION
**Pull request by gujjs**

### CODE REVIEW

File: vsdkx/addon/zoning/settings.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DEFAULT = {
+    "filter_class_ids": [],
+    "iou_thresh": (0.85,),
```
Comment (nicolexirakia) [inline vsdkx/addon/zoning/settings.py:3]: This line can be removed, it is deprecated 
```suggestion
```

File: vsdkx/addon/zoning/settings.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+DEFAULT = {
+    "filter_class_ids": [],
```
Comment (nicolexirakia) [inline vsdkx/addon/zoning/settings.py:2]: I would rename this to `class_ids` and update it also on the `processor.py` & `README.md`:
```suggestion
    "class_ids": [],
```

Comment (gujjs) [reply vsdkx/addon/zoning/settings.py:2]: I agree with you on this, but this requires more changes than that. we'd also have to make changes in settings.yaml on zoning compatible processors and all their configs, in visionx & client configs. 

If we want to do that, I presume we'd have to open separate issue or maybe even technical story on our board to track all the required changes

Comment (nicolexirakia) [reply vsdkx/addon/zoning/settings.py:2]: Let's just leave it as is for the time being and circle back about it if we see fit in the future.

**Pull request merged**